web-registry/src/hooks/useEcocredits.ts

```diff
@@ -7,17 +7,24 @@ import { ledgerRESTUri } from 'lib/ledger';
 export default function useEcocredits(address?: string): {
   credits: BatchInfoWithBalance[];
   fetchCredits: () => Promise<void>;
+  isLoadingCredits: boolean;
 } {
   const [credits, setCredits] = useState<BatchInfoWithBalance[]>([]);
+  const [isLoadingCredits, setIsLoadingCredits] = useState(true);
 
   const fetchCredits = useCallback(async (): Promise<void> => {
```

Comment:
Since loading the table still take a few seconds we do not retrigger the loader on subsequent `fetchCredits` call.